### ISSUE TITLE
massdns currently interprets cr/lf in input files as part of the domain name

### ISSUE THREAD

**Issue by eur0pa** [blechschmidt/massdns#30]
```
~ cat resolve.in
example.com

~ file resolve.in
resolve.in: ASCII text

~ cat resolve.in  | massdns -r bin/lists/good_resolvers.txt -t A -s 1
...
;; QUESTION SECTION:
example.com\x0a. IN A
```

Same behavior happens when using a list of domain names as input parameter, instead of stdin:

```
~ massdns -s 1 -t A -r bin/lists/good_resolvers.txt resolve.in

;; QUESTION SECTION:
example.com\x0a. IN A
```

Issue is not present in commit f2457e6449d02935e0f8ff17811ef89f5bd9ef1d, see my comment on this part of code: https://github.com/blechschmidt/massdns/commit/cafa6ce17c02e5132a8387508cea4cd315ae987d#r29391735

**Comment by blechschmidt** [comment]
Thank you. This should have been fixed by b838d0b2e59c813685ebed3eb88c231d55858f86.

**Comment by eur0pa** [comment]
Can confirm the new commit solved the issue
